app/javascript/components/service-request-default/index.jsx

```diff
@@ -7,13 +7,21 @@ import createSchema from './service-request-default.schema';
 // NOTE: parameters to be used as filters
 let daysAgo;
 
+const getApprovalStateCheckboxesDefault = (miqRequestInitialOptions) => {
+  var defaultStates = [];
+  for (var i=0; i < miqRequestInitialOptions.states.length; i++) {
+    defaultStates.push(miqRequestInitialOptions.states[i].value)
+  }
+  return defaultStates;
```

Comment:
Note: doing something like 
```
for (let state in miqRequestInitialOptions.states) {
    defaultStates.push(state.value)
  }
```
Didn't work in this case because of the way `miqRequestInitialOptions.states` is structured.